lib/utils/extension/src/stream.dart

```diff
@@ -5,6 +5,32 @@ extension StreamExtensionWhereNotNull<T> on Stream<T?> {
 }
 
 extension StreamExtension<T> on Stream<T> {
+  StreamSubscription<T> asyncDropListen<E>(
```

Comment:
Maybe it is needed to add some tests for this extension function to ensure the logic works as expected , since it is important which will process all background jobs...